DATA/production/qc-async/mftmchmid-tracks.json

```diff
@@ -14,9 +14,56 @@
           "query": "trackMCH:MCH/TRACKS;trackMCHROF:MCH/TRACKROFS;trackMCHTRACKCLUSTERS:MCH/TRACKCLUSTERS;mchtrackdigits:MCH/CLUSTERDIGITS;trackMFT:MFT/TRACKS;trackMFTROF:MFT/MFTTrackROF;trackMFTClIdx:MFT/TRACKCLSID;alpparMFT:MFT/ALPIDEPARAM/0?lifetime=condition&ccdb-path=MFT/Config/AlpideParam;fwdtracks:GLO/GLFWD;trackMID:MID/TRACKS;trackMIDROF:MID/TRACKROFS;trackMIDTRACKCLUSTERS:MID/TRACKCLUSTERS;trackClMIDROF:MID/TRCLUSROFS;matchMCHMID:GLO/MTC_MCHMID"
         },
         "taskParameters": {
-          "maxTracksPerTF": "600",
-          "GID" : "MFT,MCH,MID,MFT-MCH,MCH-MID,MFT-MCH-MID"
+                 "maxTracksPerTF": "600",
+                 "cutRAbsMin": "17.6",
+                 "cutRAbsMax": "89.5",
+                 "cutEtaMin": "-4.0",
+                 "cutEtaMax": "-2.5",
+                 "cutPtMin": "0.5",
+                 "nSigmaPDCA": "6",
+                 "cutChi2Max": "1000",
+                 "diMuonTimeCut": "100",
+                 "fullHistos": "0",
+                 "GID" : "MCH,MCH-MID"
+               },
+        "grpGeomRequest": {
+          "geomRequest": "Aligned",
+          "askGRPECS": "true",
+          "askGRPLHCIF": "false",
+          "askGRPMagField": "true",
+          "askMatLUT": "false",
+          "askTime": "false",
+          "askOnceAllButField": "false",
+          "needPropagatorD": "false"
         },
+        "location": "remote"
+      },
+      "TaskMUONTracksMFT": {
+        "active": "true",
+        "className": "o2::quality_control_modules::muon::TracksTask",
+        "moduleName": "QcMUONCommon",
+        "detectorName": "GLO",
+        "taskName": "MUONTracksMFT",
+        "cycleDurationSeconds": "300",
+        "maxNumberCycles": "-1",
+        "dataSource": {
+          "type": "direct",
+          "query": "trackMCH:MCH/TRACKS;trackMCHROF:MCH/TRACKROFS;trackMCHTRACKCLUSTERS:MCH/TRACKCLUSTERS;mchtrackdigits:MCH/CLUSTERDIGITS;trackMFT:MFT/TRACKS;trackMFTROF:MFT/MFTTrackROF;trackMFTClIdx:MFT/TRACKCLSID;alpparMFT:MFT/ALPIDEPARAM/0?lifetime=condition&ccdb-path=MFT/Config/AlpideParam;fwdtracks:GLO/GLFWD;trackMID:MID/TRACKS;trackMIDROF:MID/TRACKROFS;trackMIDTRACKCLUSTERS:MID/TRACKCLUSTERS;trackClMIDROF:MID/TRCLUSROFS;matchMCHMID:GLO/MTC_MCHMID"
+        },
+        "taskParameters": {
+                 "maxTracksPerTF": "600",
+                 "cutRAbsMin": "26.5",
+                 "cutRAbsMax": "89.5",
+                 "cutEtaMin": "-3.3",
+                 "cutEtaMax": "-2.5",
+                 "cutPtMin": "0.5",
+                 "nSigmaPDCA": "6",
+                 "cutChi2Max": "1000",
+                 "matchChi2MaxMFT": "45",
+                 "diMuonTimeCut": "100",
+                 "fullHistos": "0",
+                 "GID" : "MFT-MCH,MFT-MCH-MID"
```

Comment:
In order to have the possibility to evaluate the tracks matching fraction between MFT and MCH-MID, the GID parameter should be modified as
```
                 "GID" : "MFT-MCH,MCH-MID,MFT-MCH-MID"
```
This is to have MCH-MID tracks with the same kinematic cuts as the MFT-MCH-MID ones.

In case one needs to reduce the number of QC objects that are produced by the task, I would propose to eventually remove the `MFT-MCH` tracks from the selection:
```
                 "GID" : "MCH-MID,MFT-MCH-MID"
```

Reply:
@lmassacr actually I just realized that `MFT-MCH` in the GID list is useless without adding `MCH` as well, because you need the set of MCH-only plots with the same cuts as `MFT-MCH` for the comparisons.

If the number of plots becomes too big, you would need to restrict to the tracks with MID:
```
"GID" : "MCH-MID,MFT-MCH-MID"
```

Reply:
Hi @aferrero2707,
For the moment I commited the config file adding the MCH standalone tracks with the same cuts as MCH-MFT tracks. 
I will see if there are concerns from DPG regarding the number of plots, I will modify according to your proposal if this is the case. Otherwise, we can go with the current configuration now.